<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScenarioForm</class>
 <widget class="QDialog" name="ScenarioForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>986</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>290</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>698</width>
    <height>1088</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>10</width>
    <height>10</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0,0,0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <underline>true</underline>
          </font>
         </property>
         <property name="text">
          <string>Scenario</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="scenarioNameLdt"/>
       </item>
       <item>
        <widget class="QPushButton" name="AddScenarioBtn">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="CancelBtn">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Given</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="GivenAddDataBtn">
         <property name="text">
          <string>Add data array</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="GivenAddStepBtn">
         <property name="text">
          <string>Add step</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="GivenRemoveBtn">
         <property name="text">
          <string>Remove Last</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="GivenTab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>649</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>649</width>
         <height>200</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>649</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>When</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="WhenAddDataBtn">
         <property name="text">
          <string>Add data array</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="WhenAddStepBtn">
         <property name="text">
          <string>Add step</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="WhenRemoveBtn">
         <property name="text">
          <string>Remove Last</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="WhenTab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>649</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>649</width>
         <height>200</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>649</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Then</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ThenAddDataBtn">
         <property name="text">
          <string>Add data array</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ThenAddStepBtn">
         <property name="text">
          <string>Add step</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ThenRemoveBtn">
         <property name="text">
          <string>Remove Last</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="ThenTab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>649</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>649</width>
         <height>200</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>649</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinAndMaxSize</enum>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Examples</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="AddExampleBtn">
         <property name="text">
          <string>Add example</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ExampleRemoveBtn">
         <property name="text">
          <string>Remove Last</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="ExampleTab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>649</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>649</width>
         <height>200</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>649</width>
         <height>100</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
